src/components/journeys/JourneyInstancesDataTable/getTagColumns.tsx

```diff
@@ -40,8 +163,49 @@ const getTagColumns = (tagColumns: JourneyTagColumnData[]): GridColDef[] => {
           col.valueGetter(params.row as ZetkinJourneyInstance),
       });
     } else if (col.type == JourneyTagColumnType.UNSORTED) {
+      const tagsById: Record<string, ZetkinTag> = {};
+      journeyInstances
+        .flatMap((instance) => col.tagsGetter(instance))
+        .forEach((tag) => (tagsById[tag.id.toString()] = tag));
+      const uniqueTags = Object.values(tagsById).sort((t0, t1) =>
```

Comment:
This (up to and including `Object.values()`) could be broken out into a `uniqueById()` utility, either with or without the `flatMap()` part of the utility.

Reply:
duh ofcourse! this makes so much sense when you point it out :D``